documentation/identifiers.md

```diff
@@ -1,70 +1,80 @@
 # Identifiers
 
-## Why identifiers matter
+```eval_rst 
 
-Identifiers are an important part of any dataset. They let a computer uniquely identify and refer to specific grants, organisations, transactions and so-on. 
+.. admonition:: Why identifiers matter
 
-Whilst a human being may be good at recognising that:
+  Identifiers are an important part of any dataset. They let a computer uniquely identify and refer to specific grants, organisations, transactions and so-on. 
 
->INDIGO TRUST, The Indigo Trust, and indigo-trust
+  Whilst a human being may be good at recognising that "INDIGO TRUST", "The Indigo Trust", and "indigo-trust" all refer to the same organisation, computers find this a lot trickier. 
```

Comment:
It feels a bit detached in a way. Why is it trickier? Or can we maybe omit this?

Reply:
If we keep this example, please use an example of a more UK-focused grantmaker, e.g. The Big Lottery Fund, which could also be known as the Big Lottery or BLF.

Reply:
Will edit to use BLF.

Reply:
computers find it tricker because: "they don't expect the spelling variations that humans can handle."